modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+// * hosted-control-planes/index.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-concepts-personas_{context}"]
+= Glossary of common concepts and personas for hosted control planes
+
+When you use hosted control planes for {product-title}, it is important to understand its key concepts and the personas that are involved.
+
+Management cluster:: An {product-title} cluster that hosts hosted control planes and zero to many hosted clusters.
+
+Hosted cluster:: An {product-title} API endpoint that is managed by hosted control planes.
+
+Hosted control plane:: An {product-title} control plane that runs on the management cluster, which is exposed by the API endpoint of a hosted cluster. The components of a control plane include etcd, an API server, the Kubernetes controller manager, and a VPN.
+
+Hosted control plane infrastructure:: Resources on the management cluster or external cloud provider that are prerequisites to run hosted control plane processes.
+
+Management cluster infrastructure:: Network, compute, and storage resources of the management cluster.
+
+Hosted cluster infrastructure:: Network, compute, and storage resources that exist in the customer cloud account.
+
+Cluster service provider:: The user who hosts the cluster control planes and is responsible for uptime, observing the user interface for fleet-wide alerts. The cluster service provider configures the cloud account to host control planes, the user-provisioned infrastructure, which includes the host awareness of available compute, and where VMs are pulled from. This user has cluster administrator management access.
+
+Cluster service consumer:: Th user who can request control planes and workers, drive updates, or modify externalized configurations. Typically, this user does not manage or access cloud credentials or infrastructure encryption keys.
```

Comment:
```suggestion
Cluster service consumer:: The role assumed by users who can request control planes and workers, drive updates, or modify externalized configurations. Typically, this user does not manage or access cloud credentials or infrastructure encryption keys.
```

Reply:
Adding the following will also help highlight and explain the scope for the `Cluster Service Consumer`:

Can request Hosted Clusters and interact with NodePools (to upgrade,...). Users assuming this role have RBAC to CRUD HC/NodePools within some logical boundary.